src/types/sort.ts

```diff
@@ -29,3 +29,17 @@ export interface DealerSortParams {
   seed?: number
   previewId?: number
 }
+
+export enum LeadSortTypeParams {
+  NEWEST = "NEWEST",
+}
+
+export enum LeadSortOrderParams {
+  ASC = "ASC",
+  DESC = "DESC",
+}
```

Comment:
Observation: there is always only `ASC` and `DESC` sort order. Do we have specialized types for other sorts as well? Would it make sense to ditch them and use a global one?

Reply:
Yes, we have exact same for Listing, is in the same file actually `ListingSortOrderParams`. They are same, only type names are different. I suppose if I make one default one for all and make needed changes, that way make them reusable as global one. Will make the changes.

Reply:
Created generic `SortOrderParams` and switched lead na listing to use it, for listing made fallback export `SortOrderParams as ListingSortOrderParams` just as a fail-safe in case is used on other parts/repos of the project.